docs/installation_guide.md

```diff
@@ -143,6 +148,8 @@ refer to **[Configuring Harbor with HTTPS Access](configure_https.md)**.
     - **username**: username to connect harbor core database
     - **password**: password to harbor core database
     - **ssl_mode**: is enable ssl mode
+    - **max_idle_conns**: The maximum number of connections in the idle connection pool. If <=0 no idle connections are retained. The default value is 50 and if it is not configured the value is 2.
+    - **max_open_conns**: The maximum number of open connections to the database. If <= 0 there is no limit on the number of open connections. The default value is 100 for the max connections to the Harbor database. If it is not configured the value is 0.
```

Comment:
Sorry, for external database max_idle_conns and max_open_conns default values are 2 and 0.

Reply:
Fixed in https://github.com/goharbor/harbor/pull/9005/commits/da25de33a5ec397a2f49aa3acdc2af0b440c38fb.